traffic-gen/templates/testpmd.py.in

```diff
@@ -31,9 +31,9 @@ class STLS1(object):
 
 
     def get_streams (self, direction = 0, **kwargs):
-        # create multiple streams, one stream per core...
+        # create multiple streams, one stream per core generating traffic...
         s = []
-        for i in range($NUM_OF_CPUS-1):
+        for i in range($NUM_OF_TRAFFIC_CPUS):
```

Comment:
Isn't the number of created stream now larger instead of smaller?

Reply:
no, before it was $NUM_OF_CPUS-1 where [NUM_OF_CPUS](https://github.com/kiagnose/kubevirt-dpdk-checkup/blob/main/pkg/internal/checkup/checkup.go#L400) is 8 so there were 7 streams.
Now there are [NUM_OF_TRAFFIC_CPUS](https://github.com/kiagnose/kubevirt-dpdk-checkup/blob/main/pkg/internal/checkup/checkup.go#L399) which is 6 so there we'll have 6 streams.

Reply:
Where is it changed to 6?

Reply:
`NUM_OF_TRAFFIC_CPUS` is an env var we are setting on checkup package ([here](https://github.com/kiagnose/kubevirt-dpdk-checkup/blob/main/pkg/internal/checkup/checkup.go#L399)). We are already using for starting the trex on pod startup (see [here](https://github.com/kiagnose/kubevirt-dpdk-checkup/blob/main/traffic-gen/scripts/run_traffic_gen_daemon.sh#L32)). 
The env var's value is the number of CPUs (=8) minus the number of non traffic nodes (=2).